roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -134,3 +133,14 @@ def resolve_checks(names, all_checks):
         resolved.update(tag_to_checks[tag])
 
     return resolved
+
+
+def normalize(checks):
+    """Return a clean list of check names.
+
+    The input may be a comma-separated string or a sequence. Leading and
+    trailing whitespace characters are removed. Empty items are discarded.
+    """
+    if isinstance(checks, string_types):
+        checks = checks.split(',')
+    return [name.strip() for name in checks if name.strip()]
```

Comment:
works even if they pass in a hash :) Although not if it's a number or list of numbers... If you want to be really paranoid here maybe:

    return [str(name).strip() for name in checks if str(name).strip()]